src/libraries/System.Text.RegularExpressions/tests/AttRegexTests.cs

```diff
@@ -122,8 +122,8 @@ public static IEnumerable<object[]> Inputs()
                     yield return new object[] { engine, options, "aba|bab", "baaabbbaba", "(6,9)" };
                     yield return new object[] { engine, options, "(aa|aaa)*|(a|aaaaa)", "aa", "(0,2)(0,2)" };
                     yield return new object[] { engine, options, "(a.|.a.)*|(a|.a...)", "aa", "(0,2)(0,2)" };
-                    yield return new object[] { engine, options, "ab|a", "xabc", "(1,3)", "(1,2)" };
```

Comment:
I'm not sure I follow why some of these tests are being modified. I guess for tests all the changes I expected in this PR was enabling some of the ones disabled for nonBacktracking